react/typings/vtex.styleguide.d.ts

```diff
@@ -1,9 +1 @@
-declare module 'vtex.styleguide' {
-  import { ComponentType } from 'react'
-
-  export const Input: ComponentType<InputProps>
-
-  interface InputProps {
-    [key: string]: any
-  }
-}
+declare module 'vtex.styleguide'
```

Comment:
Do you need this?

Reply:
Yes, several projects are using types for the styleguide. Look: https://github.com/vtex-apps/product-list/blob/master/react/typings/vtex.styleguide.d.ts . The project doesn’t work without it

Reply:
This actually does not add any types to your project at all, that's why I asked. But I noticed this makes `tsc` stop complaining, so I'll let this pass.